pillar/spoonbill.sls

```diff
@@ -0,0 +1,22 @@
+docker:
```

Comment:
Hmm, but Spoonbill is a separated tool, maybe we should move this in a separated and independent file instead?

Reply:
The logic for Pillar data is server-based, not component-based. That's because the components need to interoperate, and it's easier to configure that in one file.

For Salt files under `salt/`, on the other hand, the logic is component-based, with any server-specific configuration stored in Pillar.

In short, you generally do not mix-and-match Pillar files, but you should be able to mix-and-match Salt files.

Reply:
When Dogsbody adds the registry backup server, we might split up `pillar/registry.sls` into `pillar/registry_common.sls` to reduce duplication, since the backup will just run the frontend without the other components, but for now one file is simplest.